develop/upload-test-results.sh

```diff
@@ -8,6 +8,7 @@ fi
 echo "uploading test results from $(pwd)"
 
 for file in *.junit.xml; do
+  if [ -e "$file" ]; then
```

Comment:
The more intuitive way to do this would be

```suggestion
shopt -s nullglob
for file in *.junit.xml; do
```

(and switch the script interpreter to bash)

For sh I kinda like the quick and easy
```suggestion
for file in *.junit.xml; do
  [ -e "$file" ] || continue
```
and then you don't have to indent anything

Reply:
Neat, thank you!